sites/mundopmmi/config/site.js

```diff
@@ -43,17 +42,18 @@ module.exports = {
   ],
   menuItems: {
     resources: [
-      { href: '/magazine', label: 'Revista' },
-      { href: '/videos', label: 'Vídeos' },
-      { href: '/white-papers', label: 'Libros Blancos' },
       { href: '/events', label: 'Eventos' },
+      { href: '/', label: 'Descargas' },
+      { href: '/videos', label: 'Vídeos' },
     ],
     userTools: [
-      { href: '/subscribe', label: 'Suscribir' },
-      { href: 'https://www.pmmimediagroup.com/pw/mundo-pmmi', label: 'Anunciar', target: '_blank' },
-      { href: '/page/about-us', label: 'Sobre nosotras' },
-      { href: '/contact-us', label: 'Contáctenos' },
-      { href: '/page/privacy-policy', label: 'Política de privacidad' },
+      { href: '/subscribe', label: 'Suscríbase' },
+      { href: '/anuncie', label: 'Anuncie' },
+      { href: '/acerca-de-nosotros', label: 'Acerca de nosotros' },
+      { href: '/contrato', label: 'Contrato' },
+      { href: '/privacidad', label: 'Privacidad' },
+      { href: '/condiciones-de-uso', label: 'Condiciones de uso' },
+      { href: '/términos-y-condiciones-de-publicidad', label: 'Términos y condiciones de publicidad' },
```

Comment:
href should be `/terminos-y-condiciones-de-publicidad`, no diacritics in urls

Reply:
Fixed